init_scripts/init-user.sh

```diff
@@ -1,11 +1,24 @@
-#!/bin/sh
-# init-user.sh - Initialize user-level requirements for our project
-# Should be set in the Vagrantfilee to run once with low privileges (non-root).
-
-# Using pipenv as our default python package manager
-# Also, we specify the version to avoid possible inconsistency
-# with future releases, and --user to reduce global fingerprint
-# on the system as a whole.
-python -m pip install -I pipenv==2020.8.13 --user
-cd /vagrant/sellotape && python -m pipenv sync
-echo "-----2ND INIT DONE-----"
+#!/bin/sh
+# init-user.sh - Initialize user-level requirements for our project
+# Should be set in the Vagrantfilee to run once with low privileges (non-root).
+
+# Add aliases to the vagrant user's .bashrc & execute permission
+echo ". /vagrant/aliases.sh" >> ~/.bashrc
+chmod +x /vagrant/aliases.sh
+
+# Using pipenv as our default python package manager
+# Also, we specify the version to avoid possible inconsistency
+# with future releases, and --user to reduce global fingerprint
+# on the system as a whole.
+python3 -m pip install -I pipenv==2020.8.13 --user
+cd /vagrant/sellotape && pipenv sync
+
+# Create required models
+pipenv run python3 sellotape_dj/manage.py migrate
+# Create default super user
+export DJANGO_SUPERUSER_PASSWORD='123456'

```

Comment:
See [here](https://github.com/ifireball/knowbali-containers-in-depth/commit/48b00daaab72352b6a2786f2c0d629045d3c11a1) for how to use a migration to create the super user - not critical for now but you may want to do it later.